### PR TITLE
fixup-mountpoints: Add Sony Z3+/Z4 (ivy)

### PR DESCRIPTION
**Pull request by Rondore**

### CODE REVIEW

File: fixup-mountpoints [hunk 1]
```diff
@@ -1796,6 +1796,16 @@ case "$DEVICE" in
...
+    "ivy")
+        sed -i \
+            -e 's /dev/block/bootdevice/by-name/system /dev/block/mmcblk0p43 ' \
+            -e 's /dev/block/bootdevice/by-name/userdata /dev/block/mmcblk0p42 ' \
+            -e 's /dev/block/bootdevice/by-name/cache /dev/block/mmcblk0p41 ' \
+            -e 's /dev/block/bootdevice/by-name/persist /dev/block/mmcblk0p31 ' \
+            -e 's /dev/block/bootdevice/by-name/diag /dev/block/mmcblk0p39 ' \
+            -e 's /dev/block/bootdevice/by-name/modem /dev/block/mmcblk0p3 ' \
+            "$@"
+
```
Comment (lnjX) [inline fixup-mountpoints:1809]: `;;` is missing

**Review by lnjX — CHANGES_REQUESTED** [review]
Please test whether the script works, before submitting a pull-request. :)

**Comment by Rondore** [comment]
Whoops, I had tested it but I ended up copying the code into the GitHub website instead of using git and missed a line. (long story) Should be good now.

File: fixup-mountpoints (hunk 1 above)
Comment (Rondore) [reply fixup-mountpoints:1809]: Whoops. I missed that line when I copied it in.

**Review by lnjX — APPROVED** [review]

**Pull request merged**